lib/p2p/Pool.ts

```diff
@@ -331,7 +355,7 @@ class Pool extends EventEmitter {
       }
       case PacketType.NODES: {
         const nodes = (packet as packets.NodesPacket).body!;
-        await this.connectNodes(nodes);
+        await this.connectNodes(nodes, false, false);
```

Comment:
I'd prefer this line without explicitly setting the boolean parameters to false since that's their default, it's just less verbose.